Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2003,6 +2009,15 @@ def lhegensim2018ml(fragment,howMuch):
                           '--conditions':'auto:run2_data'
                           },steps['TIER0']])
 
+steps['TIER0EXPRUN3']=merge([{'-s':'RAW2DIGI,L1Reco,RECO,EI,ALCAPRODUCER:SiPixelCalZeroBias+SiStripCalZeroBias+SiStripCalMinBias+SiStripCalMinBiasAAG+TkAlMinBias,DQM:@express,ENDJOB',
+                          '--process':'RECO',
+                          '--datatier':'ALCARECO,DQMIO',
```

Comment:
what is the reason for a T0 express setup to write only ALCARECO event content? IIRC, the express writes FEVT.

Reply:
this was copied from the already existing one:
https://github.com/cms-sw/cmssw/blob/988e2933bdd127ac2ae15f23a45693d28d1c61f2/Configuration/PyReleaseValidation/python/relval_steps.py#L2004-L2011
AFAIK Express step does write ALCARECO.

Reply:
All these under the standard express are alcarecos or alcaprompts:
https://github.com/dmwm/T0/blob/master/etc/ProdOfflineConfiguration.py#L246-L250

Reply:
> All these under the standard express are alcarecos or alcaprompts: https://github.com/dmwm/T0/blob/master/etc/ProdOfflineConfiguration.py#L246-L250

good, but your reference points to the Express PD, which writes `FEVT`, while this relval step is configured to write `ALCARECO`. It looks like a fix is needed.